src/modules/SdnDiag.Server/public/Get-SdnServerConfigurationState.ps1

```diff
@@ -78,8 +79,8 @@ function Get-SdnServerConfigurationState {
         Get-NetAdapterVPort | Export-ObjectToFile -FilePath $OutputDirectory.FullName -Name 'Get-NetAdapterVPort' -FileType txt -Format Table
         Get-NetAdapterVmqQueue | Export-ObjectToFile -FilePath $OutputDirectory.FullName -Name 'Get-NetAdapterVmqQueue' -FileType txt -Format Table
         Get-SdnNetAdapterEncapOverheadConfig | Export-ObjectToFile -FilePath $OutputDirectory.FullName -Name 'Get-SdnNetAdapterEncapOverheadConfig' -FileType txt -Format Table
-        Get-VMSwitch | Export-ObjectToFile -FilePath $OutputDirectory.FullName -Name 'Get-VMSwitch' -FileType txt -Format List
-        Get-VMSwitchTeam | Export-ObjectToFile -FilePath $OutputDirectory.FullName -Name 'Get-VMSwitchTeam' -FileType txt -Format List
+        Get-VMSwitch | Export-ObjectToFile -FilePath $OutputDirectory.FullName -Name 'Get-VMSwitch' -FileType json
+        Get-VMSwitchTeam | Export-ObjectToFile -FilePath $OutputDirectory.FullName -Name 'Get-VMSwitchTeam' -FileType json
```

Comment:
This converts to Json, any reason behind like that will be consumed by tools?

Reply:
Some of the properties we are collecting are not exposed when using Format-Table or Format-List, because they are objects and need to be expanded further. So previously we are missing some critical details that are important. For example:

![image](https://github.com/microsoft/SdnDiagnostics/assets/18577812/af90cdec-1c43-4489-afda-698b28312d16)

There are multiple properties under here, which will show if extension is enabled/disabled that are valuable to see, in conjunction with numerous other nested properties.